canonical-web-and-design/snapcraft.io
# Updates for dotrun

- Update to flask-base 0.4.3, so we get gunicorn 20.0.4 for dotrun
- Update flask-wtf to work with latest werkzeug
- Improve `start` script to include watching
- Use `$(hostname -I | awk '{print $1;}')` to output server IP for
  better compatibility with running inside VMs

QA
--

``` bash
dotrun clean
dotrun
```

Click on / copy the server address output in the logs, check it works in the browser to load the site.

Browse around the site, check it works as expected.
